src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -863,8 +863,11 @@ impl<'a> BinaryBuffer<'a> {
     #[inline]
     pub fn read_e_expression(self, opcode: Opcode) -> ParseResult<'a, BinaryEExpression_1_1<'a>> {
         use OpcodeType::*;
-        let (macro_address, input_after_address) = match opcode.opcode_type {
-            EExpressionWith6BitAddress => (opcode.byte as usize, self.consume(1)),
+        let (macro_id, input_after_address) = match opcode.opcode_type {
+            EExpressionWith6BitAddress => (
+                MacroIdRef::LocalAddress(opcode.byte as usize),
+                self.consume(1),
+            ),
```

Comment:
🗺️ Each arm in this `match` now returns a `MacroIdRef` instead of a `MacroAddress` (`usize`).